easybuild/easyconfigs/c/CRISPResso2/CRISPResso2-2.1.2-foss-2020b_remove_strict_deps.patch

```diff
@@ -0,0 +1,22 @@
+# unimportantly strict dependency version requirements
+# Author: Denis Kristak
+diff -ruN "CRISPResso2-2.1.2 copy/setup.py" CRISPResso2-2.1.2/setup.py
+--- "CRISPResso2-2.1.2 copy/setup.py"	2021-06-28 15:30:43.197061000 +0200
++++ CRISPResso2-2.1.2/setup.py	2021-06-28 15:36:03.124070000 +0200
+@@ -89,12 +89,12 @@
+               'Programming Language :: Cython',
+               ],
+           install_requires=[
+-              'pandas>=0.15,<=0.24',
+-              'matplotlib>=1.3.1,<=2.2.3',
++              'pandas>=0.15',
++              'matplotlib>=1.3.1',
```

Comment:
Wouldn't it be better to use a `sed` command in `preinstallopts` in the easyconfig, so we don't have to maintain this patch?

Something like this:

```python
# strip out too strict version requirements for dependencies
preinstallopts = "sed -i 's/,<=[0-9]*//g' setup.py && sed -i 's/==/>=/g' setup.py && "